src/core/diags/factory_diags.cpp

```diff
@@ -650,6 +653,34 @@ void Diags::TransmitDone(Error aError)
 
 #endif // OPENTHREAD_RADIO
 
+#if !OPENTHREAD_RADIO
+Error Diags::ProcessLinkRaw(uint8_t aArgsLength, char *aArgs[])
```

Comment:
How about adding this under the existing `radio` command? Since it already supports other radio-specific subcommands.

Reply:
Applied as suggested.